packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift

```diff
@@ -15,9 +15,9 @@ public class WalletPlugin: CAPPlugin {
             }
             let fm = FileManager.default
             let documents = fm.urls(for: .documentDirectory, in: .userDomainMask).first!
-            let path = documents.appendingPathComponent("database", isDirectory: true).path
+            let path = documents.appendingPathComponent("__storage__", isDirectory: true).path
             if !fm.fileExists(atPath: path) {
-                try fm.createDirectory(atPath: path, withIntermediateDirectories: false, attributes: nil)
+                try fm.createDirectory(atPath: path, withIntermediateDirectories: true, attributes: nil)
             }
             call.keepAlive = true
```

Comment:
Should we release this like we do on Android? I think there's a `TODO` about this on line 49